lib/migration/page_language.rb

```diff
@@ -0,0 +1,29 @@
+module Migration
+  ##
+  # This migration sets the Spotlight::Page locale to the provided one. Needed
+  # for migrating Exhibits to internationalization work.
+  class PageLanguage
+    def self.run(language)
+      new(language).run
+    end
+
+    def initialize(language)
+      @language = language
+    end
+
+    def run
+      migrate_pages
+    end
+
+    attr_reader :language
+
+    private
+
+    def migrate_pages
+      Spotlight::Page.find_each do |page|
+        page.locale = language
+        page.save
```

Comment:
Our migration to add the column defines a `default` so _I think_ the existing pages get set to the default locale (I'm happy to verify this though).

I think the thing that will cause issues for us is that the Friendly ID Slug doesn't know about the new locale scope that was added.  I suppose that this would technically address that (although I imagine we could just save/touch the record if my assumption above is true).

Another option would be to update the slugs themselves in the migration (although I obv. haven't really explored that much).  Would be happy to 🍐up on this w/ ya if you're interested.

Reply:
🍐 sounds good here. I'm not sure I understand the full implications of the friendly id scope updates.